packages/gestalt/src/Tooltip.jsdom.test.js

```diff
@@ -12,3 +15,35 @@ test('Tooltip renders', () => {
   const tree = component.toJSON();
   expect(tree).toMatchSnapshot();
 });
+
+test('Tooltip renders the hrefText when hovered', () => {
+  const { container, getByRole, getByText } = render(
+    <Tooltip
+      link={
+        <Link href="https://pinterest.com" target="blank">
+          <Text color="white" size="xs" weight="bold">
+            Learn more about logout
+          </Text>
+        </Link>
+      }
+      text="This is a tooltip"
+    >
+      <div>Hi</div>
+    </Tooltip>
+  );
+
+  fireEvent.mouseEnter(getByRole('tooltip'));
+  expect(getByText('Learn more about logout')).toBeVisible();
+  expect(container.firstChild).toMatchSnapshot();
```

Comment:
@anniejiwon can we remove these extra snapshot tests + add a test to make sure the link `https://pinterest.com` is rendered?